.github/workflows/release.yml

```diff
@@ -29,10 +29,70 @@ permissions:
 jobs:
     release:
         runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node-version: [18.x]
 
         steps:
             # Check out the repo with credentials that can bypass branch protection, and fetch git history instead of just latest commit
             - uses: actions/checkout@v3
               with:
                   token: ${{ secrets.AUTOMATION_USER_TOKEN }}
                   fetch-depth: 0
+
+            - name: Configure AWS Credentials
+              uses: aws-actions/configure-aws-credentials@v2.2.0
+              with:
+                  aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_GHA }}
+                  aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_GHA }}
+                  aws-region: us-east-1
+
+            - name: Configure git
+              run: |
+                  git config --global user.email "github-tracker-bot@taplytics.com"
+                  git config --global user.name "DevCycle Automation"
+
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
+                  cache: 'yarn'
+
```

Comment:
You'll have to add the registry url here:
https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

Reply:
ah, I wonder if that's why the npm auth was working for me locally but not on GHA.